README.md

```diff
@@ -3176,6 +3176,7 @@ _Plugin for text editors and IDEs._
 ## Go Tools
 
 - [colorgo](https://github.com/songgao/colorgo) - Wrapper around `go` command for colorized `go build` output.
+- [DarwinKit](https://github.com/progrium/darwinkit) - Build native macOS applications using Go.
```

Comment:
I don't think this is considered a **Go Tool**.

I suggest to move it to [GUI Category](https://github.com/avelino/awesome-go#gui)

Reply:
Thanks for the feedback, I moved it.